### ISSUE TITLE
UI not resizing properly and covers Repository button

### ISSUE THREAD

**Issue by quasarea** [boyter/searchcode-server#123]
I think you forgot to adjust resizing after adding Repository button

**Comment by boyter** [comment]
What size did you experience the issue with? I did try it out on a few sizes and noticed the same thing but assumed that it was such an odd size not to worry about it.

**Comment by quasarea** [comment]
I work in 1920x1200, but in many cases I do split screen, so SearchCode often gets 960x1200, while UI conflicts begin at ~1068, there is also filtering panel shift at ~1000, which could be removed especially that it downsizes only to ~777

**Comment by boyter** [comment]
Not a UI expert here, but something I will look into.

**Comment by quasarea** [comment]
Half of the screen:
* search over repositories
* filter pane takes most of the screen with results bellow it (just ~70px more and it goes left)

![2017-09-19 11_01_04-search for _screen_](https://user-images.githubusercontent.com/10334982/30587092-e19f1c58-9d29-11e7-99b3-4175ac2bb8d9.png)

![2017-09-19 11_02_40-search for _screen_](https://user-images.githubusercontent.com/10334982/30587230-51a7e2fa-9d2a-11e7-8b17-fed2bfb7949b.png)

**Comment by boyter** [comment]
Yep I am able to replicate it. Agree it is annoying. I am NOT a CSS guru however. If you know how to force bootstrap to not collapse the regions like this that would be helpful.

Otherwise ill do some checking around. I don't think it should be that hard to disable.

**Comment by quasarea** [comment]
Seems like there are four sizes adaptation:
* (max-width: 767px)
* (min-width: 768px) and (max-width: 991px)
* (min-width: 992px) and (max-width: 1199px)
* (min-width: 1200px) 
I have updated 
* `min-width: 992px` to `min-width: 922px` 
* `max-width: 991px` to `max-width: 921px`
In all files I could find it, but even after rebuild it was not applied :( will investigate tho

**Comment by boyter** [comment]
Oh the files you need to change are, in 

/src/main/resources/public/css/

The other one is just for the site https://searchcodeserver.com/ which is rather unrelated to this.
